rules/xcodeproj.bzl

```diff
@@ -163,6 +176,7 @@ def _xcodeproj_impl(ctx):
         "createIntermediateGroups": True,
         "defaultConfig": "Debug",
         "groupSortPosition": "none",
+        "settingPresets": "none",
```

Comment:
fyi to reviewers, this is described here: https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#options